src/main.py

```diff
@@ -79,6 +82,8 @@ def train_vanilla_encoder_sl(args: Namespace, train_data: list, val_data: list =
 
 def train_vtr_encoder(args: Namespace, train_data: list, val_data: list = None, test_data: list = None):
     logger.info("Training Visual Token Representation Encoder for sequence classification.")
+    ocr_flag = False if args.no_ocr else True
```

Comment:
It is already True or False, no need for explicit conversion

Reply:
And if you need opposite value then use `not args.no_ocr` or change the argument to `--ocr` 🤷‍♂️